libcudacxx/.upstream-tests/test/cuda/barrier_arrive_tx.pass.cpp

```diff
@@ -0,0 +1,124 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of libcu++, the C++ Standard Library for your entire system,
+// under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES.
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: pre-sm-90
+
+#include <cuda/barrier>
+
+#include "cuda_space_selector.h"
+#include "test_macros.h"
+#include <cstdint>              // uint and friends
+#include <utility>              // std::move
```

Comment:
Nitpick, we try to keep some order here (at least in new code).
```suggestion
#include <cstdint>              // uint and friends
#include <utility>              // std::move

#include "cuda_space_selector.h"
#include "test_macros.h"
```